thrift/storypackage.thrift

```diff
@@ -0,0 +1,10 @@
+
+struct StoryPackageItem {
+  1: required string internalPageCode,
```

Comment:
I've removed the commas and [added a recommendation](https://github.com/guardian/recommendations/pull/14)